packages/itwinui-css/src/bridge.scss

```diff
@@ -245,35 +282,117 @@
+  }
+
+  .iui-input-flex-container[data-iui-disabled='true'] {
+    // StrataKit disabled inputs don't have a different background color
+    --_iui-input-background-color: var(--iui-color-background);
+
+    // iTwinUI disabled icons don't change color, but StrataKit does. Thus, handling disabled case manually.
+    svg {
+      fill: var(--ids-color-icon-neutral-disabled);
+    }
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Label */
+
+  // StrataKit labels are slightly muted
+  .iui-input-label:not([data-iui-disabled='true']) {
+    --_iui-input-label-color: var(--ids-color-text-neutral-secondary);
   }
 
   /* ---------------------------------------------------------------------------- */
-  /* ProgressRadial */
+  /* ProgressLinear, ProgressRadial */
 
   // StrataKit progress indicators's default color is not accent
   :is(.iui-progress-indicator-linear, .iui-progress-indicator-radial) {
     --_iui-progress-indicator-color-default-progress: var(--ids-color-icon-neutral-base);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Select */
+
+  // StrataKit Select background is not --iui-color-background's global mapping.
+  .iui-select-button {
+    --_iui-field-color-background: var(--ids-color-bg-neutral-base);
+    --_iui-field-color-background-hover: var(--_iui-field-color-background);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* SideNavigation */
+
+  // iTwinUI background color (--iui-color-background-backdrop)'s global mapping doesn't look correct in this context
+  .iui-side-navigation
+    .iui-sidenav-button:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true']) {
+    background-color: var(--iui-color-background-disabled);
+  }
+
   /* ---------------------------------------------------------------------------- */
   /* Slider */
-  /* TODO: Since Slider is experimental in StrataKit, revisit this when stable on the StrataKit end */
-  /* TODO: Are minLabel and maxLabel fills/colors correct? */
+  /* TODO: Since StrataKit Slider tokens are not added, revisit this when those tokens are added on the StrataKit end */
+
+  // Track color is not --iui-color-border-accent's global mapping
+  .iui-slider {
+    --_iui-slider-track-color: var(--ids-color-bg-accent-base);
+  }
 
   // Background and border is not iui-color-background and iui-color-border-foreground respectively
   .iui-slider-thumb {
-    background-color: var(--ids-color-bg-neutral-faded);
+    background-color: var(--ids-color-bg-neutral-inverse);
     border-color: var(--ids-color-border-neutral-faded);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Surface */
+
+  .iui-surface {
+    // To match the surface similar to StrataKit's dropdown menu
+    background-color: var(--ids-color-bg-elevation-base);
```

Comment:
For Surface, we need to adjust the shadows as well. StrataKit surfaces will be flat by default (i.e. no elevation).

Maybe this should be done in the React component though.